### ISSUE TITLE
server connects to proxy but nothing works

### ISSUE THREAD

**Issue by czychuuu** [Zulfen/ZulfBungee#43]
**Describe the bug**
title
no skript reloading errors and nothing works

**To reproduce**
```
command /test:
  trigger:
    send all online proxy servers
    send this proxy server
    send this client
    send this server
    proxy kick player due to "test"
    proxy broadcast "test"
```
**Expected behavior**
everything to execute
**Screenshots**
![image](https://github.com/Zulfen/ZulfBungee/assets/60987404/0b0088dd-889e-4293-967d-6a2146d63cf1)

**Spigot server information (in this order)**
 - Plugin version [e.g. Alpha 0.7.0] 0.9.9-pre6
 - Minecraft Version: [e.g. 1.19] 1.20.4
 - Skript Version [e.g. 2.6.1] 2.8.0
 - Java Version [e.g. 8] 18

**Proxy information (in this order)**
 - Proxy Type [e.g. Bungeecord] velocity
 - Plugin Version [e.g. Alpha 0.7.0] 0.9.9-pre6
 - Minecraft Version: [e.g. 1.19] 1.7.2 - 1.20.4
 - Java Version [e.g. 8] 17

**Additional context**
Please provide your config files (censored, as in without MySQL login info) so we can check through them to see if configuration of the plugin caused the problem!
proxy https://pastebin.com/kD92C562
spigot https://pastebin.com/6RW9LQh4

**Comment by fleroviums** [comment]
Try using sockets rather than pluginmessaging if possible. I had the exact same issue and was fixed by using sockets.

**Comment by czychuuu** [comment]
> Try using sockets rather than pluginmessaging if possible. I had the exact same issue and was fixed by using sockets.

with sockets no servers connect to the proxy

**Comment by Zulfen** [comment]
> Try using sockets rather than pluginmessaging if possible. I had the exact same issue and was fixed by using sockets.

Yeah, plugin messaging is quite broken on modern Minecraft versions. Until I fix it, using sockets is reccomended :)

**Comment by fleroviums** [comment]
With plugin messaging, they only work when a player is online on the target server (as well as the one the message is coming from iirc), due to the fact that they utilise packets, which require an online player.

**Comment by Zulfen** [comment]
Will end up removing plugin messaging for now. I'll use it to automatically set up the socket stuff instead.

**Comment by Zulfen** [comment]
Fixed plugin messaging
